src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -50,6 +50,8 @@ export type MultiSelectProps = {
     flip?: boolean;
     emphasis?: TriggerEmphasis;
     enablePortal?: boolean;
+    filterable?: boolean;
```

Comment:
With this type you can end up with a filter without a label or a label without a filter. It is always better to make it impossible to create a mess. I think it is enough to have the `filterLabel`. If the label is provided it means that is filterable.
If you want to still have a boolean to control the display of the filter:
`filter?: {.enabled: true; label: string }`